docs/source/manim_directive.py

```diff
@@ -276,18 +277,20 @@ def setup(app):
 {% if not (save_as_gif or save_last_frame) %}
 .. raw:: html
 
-    <video class="manim-video" controls loop autoplay src="./{{ output_file }}.mp4"></video>
+    <video id="{{ clsname_lowercase }}" class="manim-video" controls loop autoplay src="./{{ output_file }}.mp4"></video>
 {% elif save_as_gif %}
 .. image:: /{{ filesrc_rel }}
     :align: center
+    :name: {{ clsname_lowercase }}
 {% elif save_last_frame %}
 .. image:: /{{ filesrc_rel }}
     :align: center
+    :name: {{ clsname_lowercase }}
```

Comment:
Question: I'm not following - why are these two necessary?

Reply:
Oh are these the equivalent of adding "id" on a div?

Reply:
Precisely. The `:name:` option unfortunately converts whatever is passed to lowercase, that's why I needed to add `clsname_lowercase`.